src/kixi/hecuba/weather/service.clj

```diff
@@ -1,7 +1,22 @@
 (ns kixi.hecuba.weather.service
+  (:require [clojure.java.io :as io]
+            [clojure.tools.cli :refer [parse-opts]])
   (:gen-class))
 
-(defn -main
-  "I don't do a whole lot ... yet."
-  [& args]
-  (println "Hello, World!"))
+(def cli-options
+  [["-c" "--config CONFIG" "Path to config edn file."
+    :default "config/settings.edn"]])
+
+(defn load-config [filename]
+  (with-open [r (io/reader filename)]
+    (read (java.io.PushbackReader. r))))
```

Comment:
A good question, when I've implemented (from previous reading) edn configs they've been reading this way. Not sure of the advantages over `(slurp)` in all honesty. I will certainly look though as it does make sense in the long run.